huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -631,6 +639,17 @@ func resourceComputeInstanceV2Create(d *schema.ResourceData, meta interface{}) e
 		}
 	}
 
+	// Create an instance in the shutdown state.
+	if action, ok := d.GetOk("power_action"); ok {
+		if action.(string) != "OFF" {
```

Comment:
**FORCE-OFF** should also be valid during the creation.

Reply:
thx, the FORCE-OFF has added